src/App.tsx

```diff
@@ -14,11 +14,195 @@ const goodsFromServer: string[] = [
+        visibleGoods.sort((firstWord, secondWord) => (
+          firstWord.length - secondWord.length
+        ));
+        break;
+      default:
+        (visibleGoods = [...goods]);
+    }
+
+    if (isReversed) {
+      visibleGoods.reverse();
+    }
+
+    return (
+      <div className="container">
+        <div className="row">
+          <div className="col-md-2">
+            {
+              (!isStarted
+                && (
+                  <button
+                    type="button"
+                    className="btn btn-success"
+                    onClick={this.handleToStart}
+                  >
+                    Start
+                  </button>
+                )
+              )
+            }
+          </div>
+
+        </div>
+        {(
+          isStarted
+          && (
+            <div className="row">
+              <div className="col-md-2">
+                <button
+                  type="button"
+                  className="btn btn-success"
+                  onClick={this.sortByReverse}
+                >
+                  Reverse
+                </button>
+              </div>
+
+              <div className="col-md-2">
+                <button
+                  type="button"
+                  className="btn btn-success"
+                  onClick={this.sortByAscend}
+                >
+                  Ascending
+                </button>
+              </div>
+
+              <div className="col-md-2">
+                <button
+                  type="button"
+                  className="btn btn-success"
+                  onClick={this.sortByLength}
+                >
+                  By Length
+                </button>
+              </div>
+
+              <div className="col-md-2">
+                <button
+                  type="button"
+                  className="btn btn-success"
+                  onClick={this.handleToReset}
+                >
+                  Reset
+                </button>
+              </div>
+
+              <div className="col-md-2">
+                <select
+                  className="form-select"
+                  aria-label="Default select example"
+                  name="length"
+                  value={length}
+                  onChange={(event) => {
+                    this.setState({
+                      length: Number(event.target.value),
+                    });
+                  }}
+                >
+                  <option selected>--Choose--</option>
+                  <option value="1">1</option>
+                  <option value="2">2</option>
+                  <option value="3">3</option>
+                  <option value="4">4</option>
+                  <option value="5">5</option>
+                  <option value="6">6</option>
+                  <option value="7">7</option>
+                  <option value="8">8</option>
+                  <option value="9">9</option>
+                  <option value="10">10</option>
```

Comment:
you could have created an array and use forEach so that you won't copy paste each option...